MonoGame.Framework/Input/GamePad.XInput.cs

```diff
@@ -298,6 +311,10 @@ private static GamePadState PlatformGetState(int index, GamePadDeadZone deadZone
 
         private static bool PlatformSetVibration(int index, float leftMotor, float rightMotor)
         {
+            // Make sure the player index is in range.
+            if (index < (int)PlayerIndex.One || index > (int)PlayerIndex.Four)
+                throw new InvalidOperationException();
+
```

Comment:
Can you remove the extra bounds check here in the `GamePad.XInput.cs` file?  Now that the public interface does the test this is just redundant code.